repos/fdbt-site/tests/pages/api/pricingPerDistance.test.ts

```diff
@@ -1,18 +1,25 @@
-import { getMockRequestAndResponse } from '../../testData/mockData';
+import { expectedFlatFareTicket, getMockRequestAndResponse } from '../../testData/mockData';
 import * as sessions from '../../../src/utils/sessions';
 import pricingPerDistance, { validateInput } from '../../../src/pages/api/pricingPerDistance';
 import { DistancePricingData, DistancePricing, ErrorInfo } from '../../../src/interfaces';
-import { PRICING_PER_DISTANCE_ATTRIBUTE } from '../../../src/constants/attributes';
+import {
+    MATCHING_JSON_ATTRIBUTE,
+    MATCHING_JSON_META_DATA_ATTRIBUTE,
+    PRICING_PER_DISTANCE_ATTRIBUTE,
+} from '../../../src/constants/attributes';
+import * as userData from '../../../src/utils/apiUtils/userData';
 
 describe('pricingPerDistance', () => {
     const updateSessionAttributeSpy = jest.spyOn(sessions, 'updateSessionAttribute');
     const writeHeadMock = jest.fn();
+    const s3Spy = jest.spyOn(userData, 'putUserDataInProductsBucketWithFilePath');
+    s3Spy.mockImplementation(() => Promise.resolve('pathToFile'));
 
     afterEach(() => {
         jest.resetAllMocks();
     });
 
-    it('correctly generates cap info, updates the PRICING_PER_DISTANCE_ATTRIBUTE and then redirects to /additionalPricingStructures if all is valid', () => {
+    it('correctly generates cap info, updates the PRICING_PER_DISTANCE_ATTRIBUTE and then redirects to /additionalPricingStructures if all is valid', async () => {
```

Comment:
This test file mentions cap a lot, but the page isnt for caps - its for pricing by distance. Can we update the test names and variable names please?